### PR TITLE
Minor change to include Base ASG option to docs.

### PR DESCRIPTION
**Pull request by SushrutHarness**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/continuous-delivery/deploy-srv-diff-platforms/aws/asg-tutorial.md [hunk 1]
```diff
@@ -815,7 +815,8 @@ To create the ASG infrastructure definition in an environment, do the following:
    
    You can use the same AWS connector you used when adding the AMI artifact in the Harness service. Ensure the AWS IAM user in the AWS connector credentials meets the [AWS policy requirements](#aws-policy-requirements).
 8. In **Region**, select the AWS region where you want the ASG deployed.
-9. Select **Save**.
+9. Optionally, in **Base ASG**, select an existing ASG as the base for new ASG deployments. Harness will clone the base ASG as a template to create a new ASG for the deployment.
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/deploy-srv-diff-platforms/aws/asg-tutorial.md:818]:
```suggestion
9. Optionally, in **Base ASG**, select an existing ASG as the base for new ASG deployments. Harness then clones the base ASG as a template to create a new ASG for the deployment. 
```
I'm thinking future tense is OK in this context,  but consider using present tense instead. 
https://developers.google.com/style/tense

**Review by douglas-j-bothwell — APPROVED** [review]
One optional sug otherwise LGTM

File: docs/continuous-delivery/deploy-srv-diff-platforms/aws/asg-tutorial.md (hunk 1 above)
Comment (jenniferopal) [reply docs/continuous-delivery/deploy-srv-diff-platforms/aws/asg-tutorial.md:818]: Agreed with Douglas' feedback. It reads better for the user. :)

**Review by jenniferopal — DISMISSED** [review]

**Review by jenniferopal — APPROVED** [review]
LGTM!

**Pull request merged**